contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -105,6 +108,7 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
     address target; // 2 full evm words
     uint96 amountSpent;
     address admin; // 3 full evm words
+    bool paused;
```

Comment:
ooooooph this is kinda painful... have to add a whole evm word for a single bit! Would love to see where we can cut some space. Looking at `amountSpent` and `maxValidBlocknumber` :eyes: . I think we can save that refactor for later though, once we know all the changes that are going to be made.

Reply:
this makes sense. i think uint96 is too big for possible value of amount spent.. this reminds me of writing C++ and need to pay attention to how member variables are put together.

Reply:
2**64 - 1 = 1.8446744e+19 => 18 LINK

maybe this is still too small for amount spent

Reply:
wonder if 2.8147498e+14 is large enough for the maxValidBlocknumber

Reply:
We only use amount spent to enforce the minimum spend requirement, correct? If so, we can probably make it smaller. We could have it count down from `amountLeftToSpendBeforeWithdrawEnabled` (or something less verbose but equal in significance).

Reply:
yeah we can probably trim down block number to 32 bytes. I see OCR assumes a 32 byte block number:
https://github.com/smartcontractkit/libocr/blob/master/contract2/dev/OCR2Aggregator.sol#L181

Anyway, gas golfing this is part of a separate ticket (48706) that we can do later.